sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformation.scala

```diff
@@ -58,6 +61,9 @@ case class ScriptTransformation(
 
   override def otherCopyArgs: Seq[HiveContext] = sc :: Nil
 
+  private val _broadcastedHiveConf = this.
+    sc.sparkContext.broadcast(new SerializableConfiguration(sc.hiveconf))
```

Comment:
You probably don't need broadcasting here. `SerializableConfiguration` already avoids reading XML files while deserializing `Configuration` instances.